Hestia/Hestia/frontend/Local/UITableViewControllerServerConnect.cs

```diff
@@ -93,43 +98,32 @@ public override bool ShouldPerformSegue(string segueIdentifier, NSObject sender)
 
             try
             {
-                validIp = PingServer.Check(newIP.Text, int.Parse(newPort.Text));
+                validIp = PingServer.Check(strings.defaultPrefix + newIP.Text, int.Parse(strings.defaultPort));
             }
             catch (Exception exception)
             {
                 Console.WriteLine(exception.StackTrace);
-                DisplayWarningMessage();
+                new WarningMessage("Could not connect to server", "Invalid server information", this);
+                connectButton.Selected = false;
                 return false;
             }
 
             if (validIp)
             {
-                userDefaults.SetString(newServerName.Text, strings.defaultsServerNameHestia);
-                userDefaults.SetString(newIP.Text, strings.defaultsIpHestia);
-                userDefaults.SetString(newPort.Text, strings.defaultsPortHestia);
-
                 Globals.ServerName = newServerName.Text;
-                Globals.IP = newIP.Text;
-                Globals.Port = int.Parse(newPort.Text);
-                HestiaServerInteractor serverInteractor = new HestiaServerInteractor(new NetworkHandler(Globals.IP, Globals.Port));
+                Globals.Address = strings.defaultPrefix + newIP.Text;
```

Comment:
Half of the time you are using IP and other half Address now

Reply:
The idea is that the Addres is the prefix (https:// (or http://)) + the IP. We need to split this, because we allow the user to only type the IP without the prefix. We store the complete address for easier use in the NetworkHandler constructor.